core/src/display_object/video.rs

```diff
@@ -271,6 +308,7 @@ impl<'gc> Video<'gc> {
                     }
                 }
             },
+            VideoSource::NetStream { .. } => unimplemented!(),
```

Comment:
Now that I think of it, it can get hit: if you grab a `Video` off the timeline and call `attachNetStream` on it. I'm going to make this function return silently for now.